UPGRADING-3.0.md

```diff
@@ -156,3 +156,6 @@ Upgrading From 2.x To 3.0
    * `FOS\RestBundle\View\JsonpHandler`
    * `FOS\RestBundle\View\View`
    * `FOS\RestBundle\View\ViewHandler`
+
+ * The JMS Handlers `JMSHandlerRegistry` and `JMSHandlerRegistryV2` behaviors have changed.
```

Comment:
@GuilhemN I guess this should be moved to the CHANGELOG instead since 3.0 has already been released

Reply:
Indeed, updated